### DISCUSSION TITLE
Templates and/or CSS not loading

### DISCUSSION THREAD

**Discussion by nick-nomad** [pH7Software/pH7-Social-Dating-CMS#1161] [General]
All installed but the templates are not appearing correctly in either front end or back end.
https://connectoffgrid.net/date

**Comment by alleyvandal** [comment]
Yeah, it's really screwed up. I wish somebody could fix this because it is an awesome script!

**Reply by NaoBaoDao** [reply]
ok well that does not sound like a progress.... !!

Really too bad this originally very savvy ph7 feature setup is not working anymore ... i wish i could find a similar feature list inside a matchmaking software kit ... but live and in color!!

I researched up and down the alleys... there are some kits with different commercial intentions like , cybermatch, skadate, hubpeople, ... then finally i focused flutter.js and secondly Alovoa ... maybe moodating will finally publish their promised update. 

Any more suggestion please
Nik
